src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/SearchParameters/SearchParameterFixtureData.cs

```diff
@@ -23,6 +26,7 @@ public class SearchParameterFixtureData
         public SearchParameterFixtureData()
         {
             SearchDefinitionManager = CreateSearchParameterDefinitionManager();
+            FhirPathCompiler.DefaultSymbolTable.AddFhirExtensions();
```

Comment:
@YazanMSFT Because this was missing from the tests all parameters with `resolve()` were incorrectly identified as not being supported (end eventually causing the failures in the Composite search tests)